src/network/networkd-dhcp4.c

```diff
@@ -103,6 +103,11 @@ static int link_set_dhcp_routes(Link *link) {
         if (!link->network->dhcp_use_routes)
                 return 0;
 
+        if (!link_has_carrier(link) && !link->network->configure_without_carrier)
+                /* During configuring addresses, the link losts its carrier. As networkd is dropping
```

Comment:
losts → lost (same below)

"As networkd is dropping the addresses now, let's not configure routes any more. "
→ ""As networkd is dropping the addresses now, let's not configure the routes either." ?

Reply:
Thanks. Updated.